pkg/collect/result.go

```diff
@@ -315,3 +315,52 @@ func TarSupportBundleDir(bundlePath string, input CollectorResult, outputFilenam
 
 	return nil
 }
+
+// CollectorResultFromBundle creates a CollectorResult from a bundle directory
+func CollectorResultFromBundle(bundleDir string) (CollectorResult, error) {
+	// Check directory exists
+	if _, err := os.Stat(bundleDir); os.IsNotExist(err) {
+		return nil, errors.Wrap(err, "bundle directory does not exist")
+	}
+
+	// Ensure this is indeed a support bundle directory
+	// All support bundles have a version.yaml file
+	// TODO: We might want to unmarshal the version.yaml file as well
+	_, err := os.Stat(filepath.Join(bundleDir, "version.yaml"))
```

Comment:
I don't think this check should be here.  It's just a bundle of files.  Nothing in it makes it a valid or an invalid bundle.

Reply:
Good point.